docs/products/caching/howto/upgrade-aiven-for-caching-to-valkey.md

```diff
@@ -73,7 +79,148 @@ are recycled, and your service continues to operate as the upgrade completes.
      "https://api.aiven.io/v1/project/PROJECT_NAME/service/SERVICE_NAME"
    ```
 
-</TabItem> </Tabs>
+</TabItem>
+<TabItem value="k8" label="Aiven Operator for Kubernetes®">
+:::tip
+See
+[Aiven Operator for Kubernetes configuration options for Valkey](https://aiven.github.io/aiven-operator/api-reference/valkey.html).
+:::
+
+1. [Get authenticated and authorized](https://aiven.github.io/aiven-operator/authentication.html)
+   to use the Aiven Operator for Kubernetes.
+
+1. Add the deletion policy annotation in the Aiven for Redis manifest.
+
+   1. In the manifest file of your Aiven for Redis service, for example `redis-service.yaml`,
+      add the deletion policy annotation: `controllers.aiven.io/deletion-policy: Orphan`.
+
+      ```yaml {4,5}
+      apiVersion: aiven.io/v1alpha1
+      kind: Redis
+      metadata:
+        annotations:
+          controllers.aiven.io/deletion-policy: Orphan
+        name: SERVICE_NAME
+      spec:
+        project: PROJECT_NAME
+        cloudName: CLOUD_AND_REGION_NAME
+        plan: SERVICE_PLAN_NAME
+      ```
+
+   1. Update the service by applying the configuration:
+
+      ```bash
+      kubectl apply -f redis-service.yaml
+      ```
+
+   1. Verify that the annotation has been applied:
+
+      ```bash
+      kubectl get redis SERVICE_NAME -o yaml | grep -m 1 Orphan
+       controllers.aiven.io/deletion-policy: Orphan
```

Comment:
```suggestion
```

I suspect this should be the output of the command, not the command itself :)

Reply:
ha! @ArthurFlag good catch 🙏